ser/train.py

```diff
@@ -0,0 +1,73 @@
+import torch
+from torch import optim
+import torch.nn.functional as F
+
+from ser.model import Net
+from ser.constants import PROJECT_ROOT, DATA_DIR, RESULTS_DIR
+from ser.data import load_data
+from ser.transforms import get_transforms
+
+import json
+
+def my_train(name: str, ):
+
+    print(f"Running experiment {name}")
+    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+    epochs = 2
+    batch_size = 1000
+    learning_rate = 0.01
+
+    # ----- TO-DO: save the parameters! -----
+    params = {'epochs': epochs, 'batch size': batch_size, 'learning rate': learning_rate}
+    filename =  RESULTS_DIR / "params.json"
```

Comment:
This will overwrite the parameters file for every experiment run. What you really need to do is pick an identifier for each experiment, create a directory for that experiment, and then save the results into it. That way you won't lose any old results/parameters.

I'd suggest an experiment name and timestamp as good candidates for an identifier.